docs/UserGuide.md

```diff
@@ -538,7 +635,7 @@ Action | Format and Examples
 **[Add Progress](#adding-progress-for-a-student-add--p)** | `add -p STUDENT_INDEX PROGRESS` <br> e.g., `add -p 2 completed homework`
 **[Delete Progress](#deleting-progress-from-a-student-del--p)** | `del -p STUDENT_INDEX` <br> e.g., `del -p 2`
 
-### 8.2 Lesson commands
+### 8.2 Lesson Commands
 
 |---
 Action | Format and Examples
```

Comment:
Wrong command format for `Add lesson` on line 643. should be `add -l n/LESSON_NAME [c/LESSON_CAPACITY] [p/LESSON_PRICE] [t/LESSON_TIMING]`